dev-portal/src/components/Admin/Accounts/AccountsTableColumns.jsx

```diff
@@ -0,0 +1,70 @@
+/**
+ * An AccountsTable column descriptor.
+ *
+ * @typedef {Object} AccountsTableColumns~Descriptor
+ * @property {string} id
+ *    a unique ID to distinguish this column from others
+ * @property {string} title
+ *    column title to show in the header row
+ * @property {Function} render
+ *    accepts an Account object, and returns content to be placed in the table
+ *    cell in this column
+ * @property {(Object|undefined)} ordering
+ *    ordering descriptor for this column. If absent, the user cannot order on
+ *    this column.
+ * @property ordering.iteratee
+ *    a lodash iteratee, used with `lodash.orderBy`
+ * @property {(Object|undefined)} filtering
+ *    filtering descriptor for this column. If absent, the user cannot filter
+ *    on this column.
+ * @property {string} filtering.accessor
+ *    an Account object property name on which to search
+ */
+
+export const EmailAddress = {
```

Comment:
What is this used for?

Reply:
The typedef JSDoc comment describes the format of the column descriptors below. It's currently only used in the JSDoc for `AccountsTable`.

The `EmailAddress` and other exports below are those column descriptors. They're used by consumers of `AccountsTable` to select columns to display.

Reply:
Gotcha. It's a little confusing that this is a `.jsx` file and has no actual React in it.

Reply:
Hmm. I prefer to just name all Javascript files with the same file extension (i.e. `jsx`), so that adding/removing JSX from files doesn't necessitate renaming those files.